lib/command/common/run-with-cover.js

```diff
@@ -140,7 +142,7 @@ function run(args, commandName, enableHooks, callback) {
         excludes.push(path.relative(process.cwd(), path.join(reportingDir, '**', '*')));
         matcherFor({
             root: config.instrumentation.root() || process.cwd(),
-            includes: [ '**/*.js' ],
+            includes: opts.i || [ '**/*.js' ],
```

Comment:
The default includes pattern is also hardcoded here: https://github.com/gotwarlost/istanbul/blob/bc1cbdeeacf83343955b8bf473dbd7b60cdcde10/lib/command/instrument.js#L210